src/quo2/components/wallet/account_overview/style.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.wallet.account-overview.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def account-overview-wrapper
+  {:width           335
+   :height          78
+   :align-items     :center
+   :justify-content :center})
+
+(def row {:flex-direction :row})
+
+(def account-name
+  {:margin-bottom 3
+   :color         (colors/theme-colors colors/neutral-100 colors/white)})
+
+(def reveal-icon
+  {:margin-left 4
+   :margin-top  2})
+
+(def current-value
+  {:color         (colors/theme-colors colors/neutral-100 colors/white)
```

Comment:
is this not just the default values for text anyway?

Reply:
Cleaned